ntoskrnl/mm/balance.c

```diff
@@ -39,6 +39,18 @@ static LONG PageOutThreadActive;
 
 /* FUNCTIONS ****************************************************************/
 
+static inline VOID UpdatePeakCommitment(VOID)
```

Comment:
```suggestion
static inline VOID UpdateTotalCommittedPages(VOID)
```
PeakCommitment is a consequence here.

Reply:
ok